src/actions/guides/getting-started/installing.cr

```diff
@@ -225,6 +226,26 @@ class Guides::GettingStarted::Installing < GuideAction
 
     There are other installation methods available in [Postgres CLI tools docs](https://postgresapp.com/documentation/cli-tools.html)
 
+    ### 1b. (Linux only) Password-less logins for local development
+
+    Homebrew installed PostgreSQL on macOS are configured by default to allow password-less logins.  But for Linux, if you wish to
+    use PostgreSQL w/o a password, you'll need to ensure your user is added to the `pg_hba.conf` file with `trust` method specified.
```

Comment:
The `w/o` may not be familiar to non-native English speakers so probably better to go with `without` just in case 

```suggestion
    use PostgreSQL without a password, you'll need to ensure your user is added to the `pg_hba.conf` file with `trust` method specified.
```